machine-runner/package.json

```diff
@@ -68,7 +68,8 @@
     "fast-equals": "^5.0.1",
     "ts-node": "^10.9.1",
     "typed-emitter": "^2.1.0",
-    "typescript": "^4.9.5"
+    "typescript": "^4.9.5",
+    "zod-validation-error": "^3.0.0"
```

Comment:
ah, moving to optional

Reply:
Then what is the purpose of `optional: true`? And what is the pattern for only optionally using a dependency?

Reply:
In TypeScript it can be used with `import type`
In CJS Node code, including packages use `require`. with require you can do:
```
// optionally use some module
try {
  const module = require("somemodule")
  module.someFunction();
} catch (err) {}
```